site/disk-alarms.xml

```diff
@@ -83,12 +83,18 @@ Disabling disk free space monitoring</pre>
         required to be free on the database partition. (See the description of
         <a href="/relocate.html">file locations</a> for the default location).
```

Comment:
I'm not sure from the context here what the "default location" is referring to. Is it the free disk space limit?